src/pages/MainScreen/index.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from "react"
```

Comment:
If MainScreen is being used as a layout please place it inside the layouts folder.